app/assets/stylesheets/application.scss

```diff
@@ -3,22 +3,23 @@
 // govuk_frontend_toolkit imports
 @import 'colours';
 @import 'measurements';
-@import 'typography';
 @import 'shims';
+@import 'typography';
 
 // helpers for common page elements
-@import "helpers/responsive-bottom-margin";
```

Comment:
Order was important here as `responsive-bottom-margin` is referenced in later helpers, eg https://github.com/alphagov/government-frontend/blob/master/app/assets/stylesheets/helpers/_sidebar-with-body.scss#L7

and
https://github.com/alphagov/government-frontend/blob/master/app/assets/stylesheets/helpers/_notice.scss#L3

I'm surprised the `notice` styles aren't erroring in tests.